docs/docs/configuration/server.rst

```diff
@@ -229,6 +229,86 @@ timeout
     specify a timeout in seconds for blocking operations like the
     connection attempt.
 
+Mail
+----
+
+confiurgure the format the mail notification uses in SMTP section.
+
+.. code-block:: ini
+
+    [mail]
+    language = en
+    format = plain
+    title_admin = {title}
+    title_user = Re: New comment posted on {title}
+    template =
+
+language
+	specify the language of the mail format by its `ISO 639-1<https://en.wikipedia.org/wiki/ISO_639-1>`_ (two letter)
+	code, default to `en`. The "Anonymous" will be translated in the language you
+	set. If you use the default template, the template will also be translated into
+	the language (Note that the default templates supported to be translated using
+	the option now includes: `en`, `de`, `fr`, `ja`, `zh`, `zh-CN`, `zh-TW`. You are welcome to
+	contribute by adding the translated `comment_lang.plain` and `comment_lang.html` to
+	`isso/templates`. The default `comment.plain` and `comment.html` in that directory
+	are what should be translated)
+
+format
+	specify mail format to use, available options: plain, html, multipart.
+	Default: plain
+
+title_admin
+	specify the title format of the notification email sent to the admin.
+  Default: `{title}`
+  Available variables:
+  `{title}`: The name of the isso thread.
+  `{replier}`: The author name of the comment. If the author didn't fill in the Name
+  blank, it will fallback to "Anonymous" or something else according to the language
```

Comment:
double negative and unnecessary capitalization - s/didn't fill in the Name blank/didn't fill in a name/